vscode-extension/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "relay-lsp-client",
+  "displayName": "Relay for VSCode",
+  "version": "0.0.1",
+  "description": "Relay LSP Client for VSCode",
+  "main": "./out/extension.js",
+  "activationEvents": [
+    "onLanguage:plaintext",
+    "onLanguage:javascript",
+    "onLanguage:javascriptreact",
+    "onLanguage:typescript",
+    "onLanguage:typescriptreact"
+  ],
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "Relay",
+      "properties": {
+        "relay.outputLevel": {
+          "scope": "window",
+          "type": "string",
+          "default": "quiet-with-errors",
+          "enum": ["quiet", "quiet-with-errors", "verbose", "debug"],
+          "description": "Controls what is logged to the Output Channel."
+        },
+        "relay.pathToRelay": {
+          "scope": "window",
+          "type": "string",
+          "description": "Absolute path to the relay binary. If not provided, the extension will look in the nearest node_modules directory"
+        }
+      }
+    }
+  },
+  "scripts": {
+    "watch": "tsc -w"
+  },
```

Comment:
do we have any plans integration typecheker with Github actions?

Also we probably need to add lint/prettier?

Reply:
yeah i'll add those as followups

Reply:
@alunyov any preference on including these changes in this PR vs a follow up?

Reply:
Follow up is good! Thank you!

Reply:
@alunyov I went ahead and added the following to this PR

- prettier config which is basically the same as the flow one (just had to change the parser)
- eslint config from airbnb with some slight modifications
- typecheck

All this is passing in CI

Reply:
rough timing, totally cool if you want me to pull these commits out and add them in a follow up instead.

Reply:
Oh, this is great! Thanks, it's fine to keep it here